tmt/schemas/core.yaml

```diff
@@ -0,0 +1,47 @@
+---
+
+#
+# JSON Schema for Core
+#
+# https://tmt.readthedocs.io/en/stable/spec/core.html
+#
+
+$id: /schemas/core
+$schema: https://json-schema.org/draft-07/schema
+
+definitions:
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#adjust
+  # TODO: this needs more work, it is too vague now
+  adjust:
+    oneOf:
+      - type: array
+      - type: object
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#description
+  description:
+    type: string
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#enabled
+  enabled:
+    type: boolean
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#link
+  link:
+    oneOf:
+      # TODO: revisit, maybe we want to be more strict here
+      - type: string
```

Comment:
Shouldn't this be `      - $ref: "/schemas/common#/definitions/url_path_or_fmf_id"` instead?

Reply:
I believe here the schema should say just `link` or a list of `links` defined in `common`.

Reply:
agreed